backend/settings/urls.py

```diff
@@ -23,6 +24,9 @@
 urlpatterns = [
     path("graphql/", csrf_exempt(GraphQLView.as_view(graphiql=settings.DEBUG))),
     path("api/v1/", include(router.urls)),
+    path(
+        "api/v1/owasp/api/search/algolia", csrf_exempt(algolia_search), name="api-search-algolia"
```

Comment:
WDYT about moving it into a separate `/idx/` endpoint to cover all index queries. Either `core` or `common` app?

Reply:
I guess `core`